### PR TITLE
Marking rllib_learning_tests_leela_chess_zero unstable

### PR DESCRIPTION
**Pull request by cadedaniel**
I brought up this failure to the RL team but they said it's not a release-blocking failure. We should mark it unstable so that we don't care about failures when running release tests.

https://github.com/ray-project/ray/issues/32147

### CODE REVIEW

**Comment by avnishn** [comment]
It's not unstable. There's a literal bug in the code. I found the bug yesterday and will submit a pr to unblock.

**Comment by cadedaniel** [comment]
> It's not unstable. There's a literal bug in the code. I found the bug yesterday and will submit a pr to unblock.

ok we can keep it as stable if the fix is in soon. Why is it not a release blocker since it failed due to a bug in the code?

To the release manager `stable` is an indicator of whether or not a non-infra failure should be investigated. Having more than a couple of these false-positive failures makes it really easy for a real failure to sneak in.

**Comment by cadedaniel** [comment]
e.g. https://github.com/ray-project/ray/pull/32072

**Review by kouroshHakha — APPROVED** [review]
@cadedaniel Let's mark it as unstable for now. It is an externally contributed code, and should not block release. They created a buggy code and it got somehow merged without passing the its own release test. We will internally create a fix. Release should not worry about this one at all.

**Comment by avnishn** [comment]
We've taken down the release test for this algorithm, so if its ok with you @cadedaniel I'm going to close this and OW we can reopen.

**Comment by cadedaniel** [comment]
> We've taken down the release test for this algorithm, so if its ok with you @cadedaniel I'm going to close this and OW we can reopen.

yep yep, thanks for removing it

**Pull request closed without merging**
